filebeat/processor/add_kubernetes_metadata/matchers_test.go

```diff
@@ -36,20 +37,32 @@ func TestLogsPathMatcher_InvalidSource3(t *testing.T) {
 func TestLogsPathMatcher_VarLibDockerContainers(t *testing.T) {
 	cfgLogsPath := "" // use the default matcher configuration
 	source := fmt.Sprintf("/var/lib/docker/containers/%s/%s-json.log", cid, cid)
+	if runtime.GOOS == "windows" {
+		cfgLogsPath = "D:\\containers\\logs"
```

Comment:
Here the test is testing the default value, so this variable should be the empty string.

Reply:
For this I guess that a different default has to be added in the mapper for windows.